galasa-simbank-tests/dev.galasa.simbank.tests/src/main/java/dev/galasa/simbank/tests/BasicAccountCreditTest.java

```diff
@@ -63,6 +61,16 @@ public class BasicAccountCreditTest {
      * credit the account, then checks the balance again. The test passes if the
      * final balance is equal to the old balance + the credited amount.
      * 
+     * This test works as expected however it has security issues as the credentials are hard coded.
+     * Sensitive information should not be hard coded and should be stored externally. 
+     * The coreManager could be used to register credentials and make use of methods to get the Username and Password when needed.
+     * Storing credentials externally means they can be referenced within tests instead of hard coding in every test.
+     * 
+     * The test also uses hard coded account details which is not maintainable.
+     * If the account gets deleted or details are changed, then every test using that account will need to be updated manually and individually.
```

Comment:
Can also mention that the tests that run in parrallel could collide and fail tests incorrectly